types.ts

```diff
@@ -383,13 +393,6 @@ export type yAccessorFunc = (d: any, i?: number) => number;
 
 export type Domain = number | Date | undefined;
 
-export interface VoronoiProps {
```

Comment:
@hi2dmitri FYI  there were two Voronoi props so I deleted this one and added margin to the other one.